src/iceberg/util/string_util.h

```diff
@@ -44,6 +44,11 @@ class ICEBERG_EXPORT StringUtils {
                    [](char c) { return std::toupper(c); });    // NOLINT
     return input;
   }
+
+  static bool EqualsIgnoreCase(const std::string& a, const std::string& b) {
```

Comment:
nit: We have some `operator==` definitions that use `lhs`/`rhs` as the parameters names, should we follow the same convention here?

Reply:
+1